src/main/java/duke/exception/RonException.java

```diff
@@ -0,0 +1,11 @@
+package duke.exception;
+
+/**
+ * Main exception class for chatbot
+ */
+public class RonException extends Exception {
+    @Override
+    public String toString() {
+        return "OH NO! ";
```

Comment:
You could avoid magic literals as per [this](https://nus-cs2103-ay2122s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers).

```suggestion
        String generalMessage = "OH NO! ";
        return generalMessage;
```